apps/web/src/app/admin-console/organizations/settings/components/delete-organization-dialog.component.html

```diff
@@ -0,0 +1,45 @@
+<form [formGroup]="formGroup" [bitSubmit]="submit">
+  <bit-dialog [disablePadding]="!loaded">
+    <span bitDialogTitle>{{ "deleteOrganization" | i18n }}</span>
+    <div bitDialogContent>
+      <ng-container *ngIf="!loaded">
+        <i class="bwi bwi-spinner bwi-lg bwi-spin" aria-hidden="true"></i>
+      </ng-container>
+      <ng-container *ngIf="loaded">
+        <app-callout type="warning">{{
+          "deletingOrganizationIsPermanentWarning" | i18n : organization?.name
+        }}</app-callout>
+        <p id="organizationDeleteDescription">
+          <ng-container
+            *ngIf="
+              deleteOrganizationRequestType === 'InvalidFamiliesForEnterprise';
+              else regularDelete
+            "
+          >
+            {{ "orgCreatedSponsorshipInvalid" | i18n }}
+          </ng-container>
+          <ng-template #regularDelete>
+            <ng-container *ngIf="organizationContentSummary.totalItemCount > 0">
+              {{ "deletingOrganizationContentWarning" | i18n : organization?.name }}
+              <ul>
+                <li *ngFor="let type of organizationContentSummary.itemCountByType">
+                  {{ type.count }} {{ type.localizationKey | i18n }}
+                </li>
+              </ul>
+              {{ "deletingOrganizationActiveUserAccountsWarning" | i18n }}
+            </ng-container>
+          </ng-template>
+        </p>
+        <app-user-verification formControlName="secret"> </app-user-verification>
+      </ng-container>
+    </div>
+    <div bitDialogFooter>
+      <button type="submit" bitButton bitFormButton buttonType="danger" [disabled]="!loaded">
+        {{ "deleteOrganization" | i18n }}
+      </button>
+      <button type="button" bitButton bitFormButton buttonType="secondary" (click)="cancel()">
```

Comment:
You could use `bitDialogClose` here and not need a `cancel` method in ts.

```suggestion
      <button type="button" bitButton bitFormButton buttonType="secondary" bitDialogClose>
```